src/pages/en/migrate.mdx

```diff
@@ -153,7 +153,7 @@ If you're not familiar with MDX, here are some steps you can follow to quickly c
     ```
 
 :::tip
-While you are transitioning to MDX, you may wish to [enable the legacy flag](/en/reference/configuration-reference/#legacyastroflavoredmarkdown) and include both **`.md` and `.mdx`** files, so that your site continues to function normally even before all your files have been converted. Here is one way you can do that:
+While you are transitioning to MDX, you may wish to enable the `legacy.astroFlavoredMarkdown` flag and include both **`.md` and `.mdx`** files, so that your site continues to function normally even before all your files have been converted. Here is one way you can do that:
```

Comment:
I would add whichever note previous here too. Keep the historical accuracy, but note that this no longer exists.

Reply:
```suggestion
While you are transitioning to MDX, you may wish to enable the `legacy.astroFlavoredMarkdown` flag (removed in v2.0) and include both **`.md` and `.mdx`** files, so that your site continues to function normally even before all your files have been converted. Here is one way you can do that:
```